consensus/src/counters.rs

```diff
@@ -2,74 +2,71 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use lazy_static;
-use libra_metrics::{DurationHistogram, OpMetrics};
+use libra_metrics::DurationHistogram;
 use prometheus::{Histogram, IntCounter, IntGauge};
 
-lazy_static::lazy_static! {
-    pub static ref OP_COUNTERS: OpMetrics = OpMetrics::new_and_registered("consensus");
-}
-
 lazy_static::lazy_static! {
 //////////////////////
 // HEALTH COUNTERS
 //////////////////////
 /// This counter is set to the round of the highest committed block.
-pub static ref LAST_COMMITTED_ROUND: IntGauge = OP_COUNTERS.gauge("last_committed_round");
+pub static ref LAST_COMMITTED_ROUND: IntGauge = register_int_gauge!("libra_consensus_last_committed_round","This counter is set to the round of the highest committed block.").unwrap();
 
 /// The counter corresponds to the version of the last committed ledger info.
-pub static ref LAST_COMMITTED_VERSION: IntGauge = OP_COUNTERS.gauge("last_committed_version");
+pub static ref LAST_COMMITTED_VERSION: IntGauge = register_int_gauge!("libra_consensus_last_committed_version", "The counter corresponds to the version of the last committed ledger info.").unwrap();
 
 /// This counter is set to the round of the highest voted block.
-pub static ref LAST_VOTE_ROUND: IntGauge = OP_COUNTERS.gauge("last_vote_round");
+pub static ref LAST_VOTE_ROUND: IntGauge = register_int_gauge!("libra_consensus_last_vote_round", "This counter is set to the round of the highest voted block.").unwrap();
 
 /// This counter is set to the round of the preferred block (highest 2-chain head).
-pub static ref PREFERRED_BLOCK_ROUND: IntGauge = OP_COUNTERS.gauge("preferred_block_round");
+pub static ref PREFERRED_BLOCK_ROUND: IntGauge = register_int_gauge!("libra_consensus_preferred_block_round", "This counter is set to the round of the preferred block (highest 2-chain head).").unwrap();
 
 /// This counter is set to the last round reported by the local pacemaker.
-pub static ref CURRENT_ROUND: IntGauge = OP_COUNTERS.gauge("current_round");
+pub static ref CURRENT_ROUND: IntGauge = register_int_gauge!("libra_consensus_current_round", "This counter is set to the last round reported by the local pacemaker.").unwrap();
 
 /// Count of the committed blocks since last restart.
-pub static ref COMMITTED_BLOCKS_COUNT: IntCounter = OP_COUNTERS.counter("committed_blocks_count");
+pub static ref COMMITTED_BLOCKS_COUNT: IntCounter = register_int_counter!("libra_consensus_committed_blocks_count", "Count of the committed blocks since last restart.").unwrap();
```

Comment:
Should we use label for this counter instead? example:
`pub static ref BLOCKS_COUNT: IntCounter = register_int_counter_vec!(
    "libra_consensus_blocks_count", 
    "Count of the blocks since last restart.",
    &["state"]).unwrap();`

In this way, we can have different state: state=committed, failed, success etc.

Reply:
For blocks count, there are no states, but committed transaction count has two states: failed, success. I have updated the COMMITTED_TXNS_COUNT to use a label for success, failure.